# Get current gripper position

---
float64 position
bool success
string msg
